Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json

```diff
@@ -1626,7 +1750,7 @@
 								"method": "GET",
 								"header": [],
 								"url": {
-									"raw": "{{API_URL}}/v2/claimSets?offset={{offset}}&limit={{limit}}",
+									"raw": "{{API_URL}}/v2/claimsets?offset={{offset}}&limit={{limit}}",
```

Comment:
@CSR2017 
We changed the endpoints from "claimset" to "claimSet", but most of the services were changed again to "claimset" in this PR, Is that ok?

Reply:
@jagudelo-gap 
No Juan, I got the latest and updated the tests. Will verify and update. Thanks